Feat/add translations of docs (#30)
Azordev/backend-template: README.md

> 

Reviewer: The code behind this line:

![imagen](https://user-images.githubusercontent.com/66505715/154968571-d2978513-d568-47a5-95b0-e9353099199a.png)

Are showing some extras brackets `[]`

![imagen](https://user-images.githubusercontent.com/66505715/154968640-c8ec79d1-bd9a-4926-aea7-6c58918968d3.png)

## Code Overview

> This backend's code is made using JavaScript language, and nodejs.org runtime. With a package named Express.js to create the server, routers and middleware necessary. Also PostgreSQL is used to provide a modern, open sourced and performant database layer. Lastly, the server have auxiliar packages for development and deployment to ensure is production ready, enterprise level. Typing and compile correctness by Typescript.

Reviewer: ```suggestion
This backend's code is made using JavaScript language, and Node.js runtime. With a package named Express.js to create the server, routers and middleware necessary. Also PostgreSQL is used to provide a modern, open sourced and performant database layer. Lastly, the server have auxiliar packages for development and deployment to ensure is production ready, enterprise level. Typing and compile correctness by Typescript.
```

## Code Overview

This backend's code is made using JavaScript language, and nodejs.org runtime. With a package named Express.js to create the server, routers and middleware necessary. Also PostgreSQL is used to provide a modern, open sourced and performant database layer. Lastly, the server have auxiliar packages for development and deployment to ensure is production ready, enterprise level. Typing and compile correctness by Typescript.

> As with most of node.js applications, we would want to install node and npm in our local PC in order to run the server or develop code. This folder include all the necessary files to run the server local or online, just take into account that you'll need to use a terminal as no GUI is available yet.

Reviewer 2: ```suggestion
As with most of Node.js applications, we would want to install node and npm in our local PC in order to run the server or develop code. This folder include all the necessary files to run the server local or online, just take into account that you'll need to use a terminal as no GUI is available yet.
```

Reviewer 3: En este caso creo que los nombres de paquetes deberian estar diferentes al otro texto, por ejemplo "node" o `node`

If you want to develop you have to setup a good dev environment. First of all a good text editor/IDE, I prefer to use Visual Studio Code, but you can use anything else. For the use of VSCode I provide configuration files in the hidden folder `.vscode`.

For more information about the code please check the [code overview](docs/en/CODE_OVERVIEW.md).

## Dependencies

> 

Reviewer: The code after this line, at line 174, is showing bad, I think can be better to use a unordered list

![imagen](https://user-images.githubusercontent.com/66505715/154968833-f94415fd-24a1-42a0-842d-0d688daa7bf3.png)

|          ![emmanuel-pic][]          |                                                                                                                                     |
|   [Angelica Molina][ange-github]    |![email-icon][] Email me to [angeli.molina1@gmail.com][ange-email]                                                                   |
|            ![ange-pic][]            |                                                                                                                                     |

> |    [denis Osuna][denis-github]     | ![email-icon][] Envíame un correo electrónico a [denisosuna@gmail.com][denis-email] / ![linkedin-icon][] Conectar a [my LinkedIn][denis-linkedin]  |

Reviewer: Change message to english?

```suggestion
|    [denis Osuna][denis-github]     | ![email-icon][] Email me to [denisosuna@gmail.com][denis-email] / ![linkedin-icon][] Connect to [my LinkedIn][denis-linkedin]  |
```
